pkg/clusteragent/autoscaling/workload/telemetry.go

```diff
@@ -57,6 +65,130 @@ var (
 		"Tracks the value of replicas applied by the horizontal scaling recommendation",
 		commonOpts,
 	)
+	// telemetryHorizontalScaleConstraintsMinReplicas tracks the minReplicas constraint value
+	telemetryHorizontalScaleConstraintsMinReplicas = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"horizontal_scaling_constraints_min_replicas",
+		dpaTelemetryLabels,
+		"Tracks the minReplicas constraint value from DatadogPodAutoscaler.spec.constraints",
+		commonOpts,
+	)
+	// telemetryHorizontalScaleConstraintsMaxReplicas tracks the maxReplicas constraint value
+	telemetryHorizontalScaleConstraintsMaxReplicas = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"horizontal_scaling_constraints_max_replicas",
+		dpaTelemetryLabels,
+		"Tracks the maxReplicas constraint value from DatadogPodAutoscaler.spec.constraints",
+		commonOpts,
+	)
+
+	// telemetryVerticalScaleConstraintsContainerCPURequestMin tracks minimum CPU request constraint per container
+	telemetryVerticalScaleConstraintsContainerCPURequestMin = telemetry.NewGaugeWithOpts(
```

Comment:
I use different metrics for CPU and memory because the unit is different and with datadog metrics it is not a good practice to have the same metric but with different units.